### PR TITLE
wider dub versions allowed

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by John-Colvin** [comment]
It would also be useful if you could make a new release with this change

File: dub.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	"targetPath": "bin",
 	"dependencies": {
 		"derelict-pq": "~>3.0.0-alpha.2",
-		"vibe-d:data": "~>0.7.27-rc.2"
+		"vibe-d:data": ">=0.7.27 <0.9.0"
```
Comment (denizzzka) [inline dub.json]: 0.9.0 is version from future?

**Comment by denizzzka** [comment]
Can we add CI check for vibe versions 0.7.27, 0.8.xx, 0.9.xx?

File: dub.json (hunk 1 above)
Comment (John-Colvin) [inline dub.json]: yes, but we have `<` there meaning we reject it (who knows what changes might be in that release)

**Comment by John-Colvin** [comment]
> Can we add CI check for vibe versions 0.7.27, 0.8.xx, 0.9.xx?
In theory yes, but I'm not there's a neat way

File: dub.json (hunk 1 above)
Comment (denizzzka) [inline dub.json]: Ok

**Pull request merged**